src/webview/webViewEventHandlers.js

```diff
@@ -81,6 +83,11 @@ type MessageListEventLongPress = {|
   messageId: number,
 |};
 
+type MessageListEventLongPressUrl = {|
```

Comment:
You might want to consider adding the `href` parameter to `MessageListEventLongPress` and add another `target` as `url`/`link`.

Reply:
That seems to be a good idea. Thanks! :slightly_smiling_face: